packages/legend-studio/src/stores/editor-state/element-editor-state/mapping/MappingTestState.ts

```diff
@@ -274,10 +274,14 @@ export class MappingTestRelationalInputDataState extends MappingTestInputDataSta
 
   get runtime(): Runtime {
     const datasourceSpecification = new LocalH2DatasourceSpecification();
-    datasourceSpecification.setTestDataSetupSqls(
-      // NOTE: this is a gross simplification of handling the input for relational input data
-      [this.inputData.data],
-    );
+    if (this.inputData.inputType === RelationalInputType.SQL) {
+      datasourceSpecification.setTestDataSetupSqls(
+        // NOTE: this is a gross simplification of handling the input for relational input data
+        [this.inputData.data],
+      );
+    } else {
```

Comment:
let's do `else if` and have the clause `else` for throwing an `Unsupported...` error